### PR TITLE
JS Capstone: Comments

### PR DESCRIPTION
**Pull request by Kwaku28**
## This part of the project is to configure the comments section

The functions added include the following:

1. A function to add new comments
2. A function to post the new comments to the API server
3. A function to retrieve and display the comments
4. A function to count the number of comments added

Thank you🤝

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import postComments from './module/postComments.js';
 import newComment from './module/newComment.js';
 import countComments from './module/countComments.js';
 import fetchList from './module/fetchList.js';
+// import countItems from './module/countItems.js';
+import { updateLikes } from './module/aboutLikes.js';
```
Comment (KUBAHO3) [inline src/index.js]: Please consider removing all the comments code as it makes your code looks messy

**Review by KUBAHO3 — COMMENTED** [review]
- ✅All linters ahve passed succesfuly 
- ✅Best practices are followed 
- ✅ Nice descriptive review added

Please Consider the comments below and update the codes as required, Thank you!

File: src/index.js (hunk 1 above)
Comment (Kwaku28) [reply src/index.js]: Thank you for the review. As suggested, I have removed the comments from the code.

**Comment by KUBAHO3** [comment]
Awesome! Thank you for considering making changes on the mentioned slice of code.
Now your project is **Approved!**
approved

**Pull request merged**